### ISSUE TITLE
Headers in README.md sample are undefined

### ISSUE THREAD

**Issue by bterlson** [Azure/azure-storage-js#10]
The sample code in the README attempts to get headers, but the result is undefined. It also doesn't appear that `headers` is a valid property on `downloadBlockBlogResponse`.

Here's the sample code for reference:

```js
console.log(`[headers]:${downloadBlockBlobResponse.headers}`);
```

**Comment by XiaoningLiu** [comment]
Thanks, the headers property has been moved. Will update.

I'm guessing whether it's a good way to put a sample in readme.md, because it's out of testing, we may forget to updating the sample when API changes, just like this issue.

**Comment by bterlson** [comment]
One possibility is you can turn the sample into a test and put a comment in the test file that says any updates to the test should also be reflected in the readme.

**Comment by XiaoningLiu** [comment]
Resolved in release 10.1.0-preview, thanks!
